packages/rhash/build.sh

```diff
@@ -2,14 +2,14 @@ TERMUX_PKG_HOMEPAGE=https://github.com/rhash/RHash
 TERMUX_PKG_DESCRIPTION="Console utility for calculation and verification of magnet links and a wide range of hash sums"
 TERMUX_PKG_LICENSE="MIT"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION=1.4.1
+TERMUX_PKG_VERSION=1.4.2
 TERMUX_PKG_SRCURL=https://github.com/rhash/RHash/archive/v$TERMUX_PKG_VERSION.tar.gz
-TERMUX_PKG_SHA256=430c812733e69b78f07ce30a05db69563450e41e217ae618507a4ce2e144a297
+TERMUX_PKG_SHA256=600d00f5f91ef04194d50903d3c79412099328c42f28ff43a0bdb777b00bec62
 TERMUX_PKG_DEPENDS="openssl"
 TERMUX_PKG_CONFLICTS="librhash, rhash-dev"
 TERMUX_PKG_REPLACES="librhash, rhash-dev"
 TERMUX_PKG_BUILD_IN_SRC=true
-TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-static --enable-lib-static --enable-lib-shared"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="--disable-static --enable-lib-static --enable-lib-shared --cc=clang"
```

Comment:
Pass the proper compiler tuple there, either `$CC` or `${TERMUX_HOST_PLATFORM}-clang`. You are cross-compiling.